### ISSUE TITLE
Button in telegram for reupdate the tradesv3.sqlite

### ISSUE THREAD

**Issue by Vildnex** [freqtrade/freqtrade#3694]
## Describe your environment
(if applicable)

  * Operating system: Ubuntu Server 20.04
  * Python Version: 3.8.2 (`python -V`)
  * CCXT version: 1.32.7 (`pip freeze | grep ccxt`)
  * Freqtrade Version: 2020.7 (`freqtrade -V` or `docker-compose run --rm freqtrade -V` for Freqtrade running in docker)

## Describe the enhancement

I'm running the bot on a real money account using Binance and in some cases, it happens to me a few interesting things.
1. some times it's gets stuck into selling point (**) and it's never gonna sell until I will go manually and close the order.
2. because some amount is different. Basically, because whatever reason in my Binance account I don't have a pair (currently is FET/USDT) which the bot still counts as an existing one. 
3. Also, I have some times errors like this:
`ERROR - Unable to place a stoploss order on exchange. Could not create stop_loss_limit sell order on market TOMO/USDT. Tried to sell amount 4.0 at rate 1.2132. Message: binance {"code":-1013,"msg":"Filter failure: MIN_NOTIONAL"}`
And in order to solve this, I need to go again in Binance to close it manually 

Some times during the process of closing or selling manually the bot is going to get confused and it will display to me some pairs which it thinks that those pairs are there, even those are not.

In short, I would like it if it's possible a button in telegram which will **update** my **tradesv3.sqlite** with the real values from Binance. The only workaround in order to fix my problem for the moment been to delete and recreate that file again.

**Comment by xmatthias** [comment]
Will assume this as answered by the above comment.

Feel free to comment below / reopen if you still have questions regarding this.
